CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # CHANGELOG
 
 ## HEAD
+* Update file dependencies of the new structure
```

Comment:
I remove this point because everybody knows that you need to update file refecences/dependencies, if you change the filename